rct229/rulesets/ashrae9012019/ruleset_functions/get_hvac_systems_serving_zone_health_safety_vent_reqs.py

```diff
@@ -0,0 +1,72 @@
+from rct229.rulesets.ashrae9012019.data.schema_enums import schema_enums
+from rct229.rulesets.ashrae9012019.ruleset_functions.get_list_hvac_systems_associated_with_zone import (
+    get_list_hvac_systems_associated_with_zone,
+)
+from rct229.utils.jsonpath_utils import find_all
+from functools import reduce
+from operator import add
+
+LightingSpaceOptionsG37 = schema_enums["LightingSpaceOptions2019ASHRAE901TG37"]
+VentilationSpaceOptions = schema_enums["VentilationSpaceOptions2019ASHRAE901"]
+
+LIGHTING_SPACE_TYPES_MATCH_REQ = [
+    LightingSpaceOptionsG37.MANUFACTURING_FACILITY_DETAILED_MANUFACTURING_AREA,
+    LightingSpaceOptionsG37.MANUFACTURING_FACILITY_EQUIPMENTROOM,
+    LightingSpaceOptionsG37.HEALTHCARE_FACILITY_EXAM_TREATMENT_ROOM,
+    LightingSpaceOptionsG37.HEALTHCARE_FACILITY_MEDICAL_SUPPLY_ROOM,
+    LightingSpaceOptionsG37.HEALTHCARE_FACILITY_NURSERY,
+    LightingSpaceOptionsG37.HEALTHCARE_FACILITY_OPERATING_ROOM,
+    LightingSpaceOptionsG37.HEALTHCARE_FACILITY_PATIENT_ROOM,
+    LightingSpaceOptionsG37.HEALTHCARE_FACILITY_RECOVERY_ROOM,
+    LightingSpaceOptionsG37.MUSEUM_RESTORATION_ROOM,
+    LightingSpaceOptionsG37.PHARMACY_AREA,
+    LightingSpaceOptionsG37.EMERGENCY_VEHICLE_GARAGE,
+    LightingSpaceOptionsG37.CONFINEMENT_CELLS,
+]
+
+VENTILATION_SPACE_TYPES_MATCH_REQ = [
+    VentilationSpaceOptions.OUTPATIENT_HEALTH_CARE_FACILITIES_CLASS_1_IMAGING_ROOMS,
+    VentilationSpaceOptions.ANIMAL_FACILITIES_ANIMAL_IMAGING_MRI_CT_PET,
+]
+
+
+def get_hvac_systems_serving_zone_health_safety_vent_reqs(rmi):
```

Comment:
The RDS has the argument being `RMR`. So the RDS should be updated to match this implementation.